packages/web3-eth/test/integration/defaults.transactionBlockTimeout.test.ts

```diff
@@ -19,62 +19,60 @@ import { DEFAULT_RETURN_FORMAT } from 'web3-utils';
 import { Web3PromiEvent } from 'web3-core';
 import { TransactionReceipt } from 'web3-types';
 import { TransactionBlockTimeoutError } from 'web3-errors';
-import { SendTransactionEvents, Web3Eth } from '../../src';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Web3 } from 'web3';
+import { Web3Account } from 'web3-eth-accounts';
+import { SendTransactionEvents } from '../../src';
 
 import {
 	closeOpenConnection,
-	createTempAccount,
 	getSystemTestProvider,
 	isSocket,
 	itIf,
 	waitForOpenConnection,
+	createLocalAccount,
+	sendFewSampleTxs,
 } from '../fixtures/system_test_utils';
 
-import { sendFewTxesWithoutReceipt } from './helper';
-
 const MAX_32_SIGNED_INTEGER = 2147483647;
+const gas = 21000;
 /* eslint-disable jest/no-standalone-expect */
 describe('defaults', () => {
-	let web3Eth: Web3Eth;
-	let eth2: Web3Eth;
+	let web3: Web3;
 	let clientUrl: string;
+	let account1: Web3Account;
+	let account2: Web3Account;
 
-	beforeEach(() => {
+	beforeEach(async () => {
```

Comment:
```suggestion
	beforeEach(() => {
```

`await` is never used in this code block?

Reply:
done